buf.gen.all.yaml

```diff
@@ -1,4 +1,11 @@
 version: v1
+managed:
+  enabled: true
+  java_multiple_files: true
+  java_package_prefix:
```

Comment:
This is a really neat feature of buf. However, did you know that this is supported for C# and go as well? We could even get rid of the `option go_package` in all of the protos if we would add a `go_package_prefix` next to this - it's mentioned in [this docs page](https://docs.buf.build/tour/use-managed-mode#remove-go_package). We don't have to do it now since I have a feeling this would be a bigger change.